### ISSUE TITLE
Use minus sign instead of unary minus in most places

### ISSUE THREAD

**Issue by jbphet** [phetsims/number-line-operations#9]
@amanda-phet requested in the design doc that we use a minus sign instead of the unary minus symbol in most places, because the combination of regular minus signs and unary minus signs can end up looking a little weird.  Here is an example from a recent [dev version](https://phet-dev.colorado.edu/html/number-line-operations/1.0.0-dev.4/phet/number-line-operations_en_phet.html):

![image](https://user-images.githubusercontent.com/3082831/85894195-8663c000-b7b1-11ea-985c-075910641da3.png)

When we discussed it, we decided to leave the unary minus in the number picker, since that looks pretty good.

**Comment by jbphet** [comment]
I'm logging this as a GitHub issue because I think it requires some discussion and refinement.  I started making all of the minus signs into `MathSymbols.MINUS` and realized that there are several usages of `MathSymbols.UNARY_MINUS` in the number-line-common repo, so changing these would affect number-line-integers and number-line-distance too.  And, after experimenting with it some, I think the unary minus sign looks better in many of the places where it is currently used, such as for tick mark labels, point labels, and the range selector.  I'd like to propose that we use the unary minus any time the numbers appear on their own, but use the binary minus (`MathSymbols.MINUS` in our code base) whenever the number is in an expression or other context where it can appear adjacent to another plus or minus sign.  So, for example, the case shown above would become:

![image](https://user-images.githubusercontent.com/3082831/85899407-3be74100-b7bb-11ea-9e44-da8a090b8ca1.png)

...but point and tick mark labels would continue to look like this:

![image](https://user-images.githubusercontent.com/3082831/85899607-89fc4480-b7bb-11ea-9cf4-3977134cbc23.png)

@amanda-phet - over to you for consideration.  I'll implement what I've described here, and it will be available on master within the hour.  Let me know if you'd like a dev version to try out.
